config/base.py

```diff
@@ -308,6 +308,10 @@ def get_environment_variable(var_name, json_environment_vars=json_environment_va
 
 EMAIL_BACKEND = get_environment_variable("EMAIL_BACKEND")
 SENDGRID_API_KEY = get_environment_variable("SENDGRID_API_KEY")
+ADMINS = get_environment_variable("ADMINS")
+# Expecting a space delimited string of emails like "jane@wevote.org" or "jane@wevote.org bill@wevote.org"
+if ADMINS:
+    ADMINS = [[email.split('@')[0], email] for email in ADMINS.split()]
```

Comment:
Hi @SailingSteve, I learned the hard way on a large production system not to use email alerts for specific individual errors. Perhaps we could store in the database a count of errors seen in the system per day, and then email ADMINS daily the error count with a reminder to check Splunk? I look forward to discussing with you.